indexer/services/vulcan/src/handlers/order-remove-handler.ts

```diff
@@ -531,6 +551,53 @@ export class OrderRemoveHandler extends Handler {
     return sizeDelta.toFixed();
   }
 
+  /**
+   * Should be called when an OrderRemove message is received for a non-existent order.
+   * This can happen when the order was not found in redis because the initial order
+   * placement message wasn't received.
+   *
+   * @param canceledOrder
+   * @param orderRemove
+   * @param perpetualMarket
+   * @protected
+   */
+  protected createSubaccountWebsocketMessageFromOrderRemoveMessage(
+    canceledOrder: OrderFromDatabase | undefined,
```

Comment:
If `canceledOrder` is not undefined, you can populate more of the order contents, including price / size / etc.